packages/devtools/client/composables/state-routes.ts

```diff
@@ -0,0 +1,28 @@
+// server-routes Tab
+export function useServerRoutesState() {
+  return useState<string>('server-routes:state')
```

Comment:
Maybe we should use `useSessionStorage` to preserve the state across refreshes?

Reply:
Yes. that make sense. thank you